apm-agent-plugins/apm-awslambda-plugin/src/main/java/co/elastic/apm/agent/awslambda/helper/AbstractLambdaTransactionHelper.java

```diff
@@ -80,6 +88,18 @@ public Transaction startTransaction(I input, Context lambdaContext) {
         return null;
     }
 
+    private String getFaasId(Context lambdaContext) {
+        if (functionArn == null) {
+            functionArn = lambdaContext.getInvokedFunctionArn();
+            String[] arnSegments = functionArn.split(":");
+            if (arnSegments.length > 6) {
```

Comment:
```suggestion
            if (arnSegments.length > 7) {
```

This is a bug, needs to be 7, as the standard Lambda function ARN without version segment contains 7 segments.

Reply:
I wonder how stable this is. Is there a documented specification regarding the ARN structure?